.travis.yml

```diff
@@ -51,6 +56,10 @@ script:
 - JS_WARNINGS_AMOUNT_UPPER_LIMIT=37
 - eslint ./src --max-warnings $JS_WARNINGS_AMOUNT_UPPER_LIMIT
 
+after_success:
+  # Upload code coverage results to codecov.io, see https://github.com/codecov/codecov-bash for more information
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
You're not in the same directory you were in when you ran the code coverage, is this a problem?

Reply:
Nope, their script just works by searching recursively. This will also help us uf we will want to run coverage in the `/envs` or any other of the root subdirs in future.

Reply:
Searching recursively but we're "below" the root folder